assignment/assignment.hs

```diff
@@ -16,7 +16,12 @@ import Data.Char(intToDigit)
 --
 -- Implement a version vcalled `myreverse` using `foldl`.
 
--- myreverse :: [a] -> [a]
+
+prepend list element = element : list
+--myreverse :: [a] -> [a]
+myreverse = foldl (prepend) []
+
```

Comment:
Very nice. And `prepend` is a good name for that function.

A minor style point: you don't need the parens on line 22.